public/js/index.js

```diff
@@ -34,291 +34,314 @@ const clearBtn = document.querySelector(".create-lesson-clear");
 
 // event listener setup
 formElement.addEventListener("submit", function (e) {
-  e.preventDefault();
-  addLesson();
+	e.preventDefault();
+	addLesson();
 });
+
+window.onload = handleClearBtn;
```

Comment:
Instead of running this function on a window onload event, maybe it would be better to attach an event listener to the `form` element and listen for mouse/keyup events on the `form` child elements such as the title `input`. By attaching the event listener to the `form` element it would save us having to attach it to the title input & content editor elements - this is called [event delegation](https://javascript.info/event-delegation) in JS.

At the moment, when the  `window.onload` event is triggered this will invoke `handleClearBtn()` which in turn goes on to set `onmouseover` handlers.

Reply:
Let me know if this is confusing!